content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+- scip-clang is based on Clang 16 instead of Clang 11.
+  It consumes Clang as a library rather than as a fork,
+  making it easy to update the version of Clang used in the future. 
+
+Some other minor quality-of-life improvements include:
+
+- scip-clang infers paths to standard headers for GCC and Clang
+  from the compilation database without requiring extra command-line flags.
+- scip-clang binaries are available for both Linux and macOS.
+- Despite supporting more features, thanks to SCIP, index sizes
+  (both compressed and uncompressed) are about
+  10%-20% the size of the corresponding LSIF indexes.
+  This translates to faster uploads, a lower likelihood of upload errors,
+  and reduced risk of out-of-memory when the Sourcegraph backend
+  processes an index.
+- scip-clang uses incremental parsing for compilation databases,
+  reducing the risk of out-of-memory errors on ingestion.
+
+If you are a Sourcegraph customer using C or C++,
+please [try out scip-clang](https://github.com/sourcegraph/scip-clang),
+and let us know if you run into issues,
+or if you have feedback for improvement.
+
+As with our other indexers, open source maintainers are also welcome
+to use scip-clang to index their projects
+and upload indexes to Sourcegraph.com
+to benefit from precise code navigation for C and C++.
+
+## A word about performance
+
+scip-clang requires a traversal of the abstract syntax tree
+after type-checking the code, so that type information is available.
+
+Compared to a baseline of type-checking all translation units in parallel,
+scip-clang takes about 30% more time.
+
+In the future, we will be able to reduce this overhead in two ways:
+- [Optimizing the release build](https://github.com/sourcegraph/scip-clang/issues/27)
+  by using well-known techniques such as ThinLTO and PGO.
+- [Parallelizing the index merging step](https://github.com/sourcegraph/scip-clang/issues/139),
+  which combines information about forward declarations
+  and project-external symbols across translation units.
+  Depending on codebase size and core count,
+  about 10%-40% of the overall indexing time is spent in index merging.
+
+Compared to our indexer lsif-clang, scip-clang takes about 80%-90%
+more time to index the same codebase. The reason for this is that lsif-clang
+avoids type-checking many declarations, such as compiler-synthesized ones,
+since they are not indexed.
+We're interested in [surfacing information about synthesized declarations](https://github.com/sourcegraph/scip/issues/117#issuecomment-1422654456) in the future,
+so it may not make sense to perform this same optimization in scip-clang,
+only to remove it at a later stage.
+
+We hope that the improved robustness and
+higher quality code navigation with scip-clang
+make up for the loss in performance for current lsif-clang adopters.
+
+## What's next
+
+In the coming months, we're looking to bring scip-clang up to parity
```

Comment:
I'd inverse this section. In the coming months we'll be adding support for cross-repo and additional complex language features like... that will bring us to parity alongside...